app-near/src/app/api/ngcform/email/route.ts

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { TRPCError } from "@trpc/server";
+import { type NextRequest } from "next/server";
+import { handleCarbonFootprintEmail } from "~/server/carbon-footprint/handleCarbonFootprintEmail";
+import { getHTTPStatusCodeFromError } from "@trpc/server/http";
+import { ErrorCode } from "~/types/enums/error";
+
+const anonymize = (data: Record<string, any>) => ({ ...data, email: "" });
+
+export async function POST(req: NextRequest) {
+  try {
+    return handleCarbonFootprintEmail(req);
+  } catch (error) {
+    if (error instanceof TRPCError) {
+      return new Response(
+        JSON.stringify({
+          code: error.code,
+          message: error.message,
+          data: error.cause ?? null,
+        }),
+        { status: getHTTPStatusCodeFromError(error) },
+      );
+    }
+
+    const body = await req.json();
+    let message = body;
+
+    if (typeof body === "object" && !Array.isArray(body)) {
+      message = JSON.stringify(anonymize(body as Record<string, any>));
+    }
+
```

Comment:
:green_circle:  Autant faire cela non ? : 
```suggestion
    let body = await req.json();

    if (typeof body === "object" && !Array.isArray(body)) {
      body = JSON.stringify(anonymize(body as Record<string, any>));
    }
```